crates/holochain/src/core/workflow/countersigning_workflow.rs

```diff
@@ -115,14 +117,27 @@ pub(crate) async fn countersigning_workflow(
 
     // For each complete session send the ops to validation.
     for (agents, ops, actions) in complete_sessions {
-        incoming_dht_ops_workflow(space, sys_validation_trigger.clone(), ops, false).await?;
+        let non_enzymatic_ops: Vec<_> = ops
+            .into_iter()
+            .filter(|(_hash, dht_op)| !dht_op.enzymatic_countersigning_enzyme().is_some())
+            .collect();
+        incoming_dht_ops_workflow(
```

Comment:
Might be best to avoid calling this workflow if the `non_enzymatic_ops` is empty?